cli/templatecreate.go

```diff
@@ -195,14 +241,22 @@ func createValidTemplateVersion(cmd *cobra.Command, client *codersdk.Client, org
 
 		// parameterMapFromFile can be nil if parameter file is not specified
 		var parameterMapFromFile map[string]string
-		if parameterFile != "" {
+		if args.ParameterFile != "" {
```

Comment:
I think we should remove the `ParameterFile` in favor of a `--var=asd=asd` approach. Separate issue, but I think it conflates some of this logic.

Reply:
The file logic needs to be looked at for sure. I kept it as is for now.